docs/en/setup/service-agent/java-agent/Bootstrap-plugins.md

```diff
@@ -19,4 +19,3 @@ The plugin of JDK HttpURLConnection depended on `sun.net.*`. When using Java 9+,
 
 For more information 
 1. [JEP 403: Strongly Encapsulate JDK Internals](https://openjdk.org/jeps/403)
-2. [A peek into Java 17: Encapsulating the Java runtime internals](https://blogs.oracle.com/javamagazine/post/a-peek-into-java-17-continuing-the-drive-to-encapsulate-the-java-runtime-internals)
```

Comment:
I think you made a mistake to remove this line?

Reply:
> I think you made a mistake to remove this line?

No, the CI thiks this line is a deadlink, though it can be access in browser

Reply:
The blog is there. What is the deadlink check saying?

Reply:
You should not remove the docs, just because a CI took says otherwise.

Reply:
> You should not remove the docs, just because a CI took says otherwise.

OK, I will rollback it, thank you!